less/footer.less

```diff
@@ -1 +1,20 @@
-// Footer Styles
\ No newline at end of file
+// Footer Styles
+html {
+    font-size: 62.5%;
+}
+
+footer {
+    @media @mobile {
+        width: 100%;
+    }
+    border: 1px solid  @border-color-for-main-content;
+    color: @footer-text-color;
+    background-color: @footer-background-color;
+    width: 70%;
+    display: flex;
```

Comment:
You have a mixin that does these three common things. `.center()`, don't forget about using these very useful mixins!